src/parser/jobs/pld/About.js

```diff
@@ -0,0 +1,21 @@
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+
+import CONTRIBUTORS from 'data/CONTRIBUTORS'
+import CoreAbout from 'parser/core/modules/About'
+
+export default class About extends CoreAbout {
+	description = <Fragment>
```

Comment:
As per pin in `#development` - I'm not going to block on this, but have a look into changing up the description a little.

Reply:
Will do.
As I mentioned in PR and in chat, this was more or less whipped up in a short evening before sleep, so I just copied something to get something semi-sensible in place.